src/win/util.c

```diff
@@ -1135,6 +1136,13 @@ int uv_getrusage(uv_rusage_t *uv_rusage) {
     return uv_translate_sys_error(GetLastError());
   }
 
+  ret = GetProcessMemoryInfo(GetCurrentProcess(),
+                             &memCounters,
+                             sizeof(PROCESS_MEMORY_COUNTERS));
```

Comment:
The current code isn't incorrect but it would be if the type of `memCounters` changes.  That's why it's generally a good idea to take the size of the variable instead of its type.